mycroft/skills/core.py

```diff
@@ -516,7 +517,7 @@ def shutdown(self):
         shutdown all processes and operations in execution.
         """
         # Store settings
-        self.settings.store()
+        self.settings.store(force=True)  # Force saving of settings
```

Comment:
Let's drop the force.  I'm nervous about forcing a disk write by every single skill during shutdown scenarios, which is when the rug is most likely to be pulled out from under you -- impatient user unplugs device, script timeout passes and a kill gets executed, etc.